### PR TITLE
docs: created a new folder and sidebar for v4.1

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

File: docs/main/releases.md [hunk 1]
```diff
@@ -2,39 +2,106 @@
...
+## Watch Items and Known Issues
+
+### Watch Items and Known Issues - Local Storage
+
+Local PV ZFS / Local PV LVM on a single worker node encounters issues after upgrading to the latest versions. The issue is specifically associated with the change of the controller manifest to a Deployment type, which results in the failure of new controller pods to join the Running state. The issue appears to be due to the affinity rules set in the old pod, which are not present in the new pods. As a result, since both the old and new pods have relevant labels, the scheduler cannot place the new pod on the same node, leading to scheduling failures when there's only a single node.
+The workaround is to delete the old pod so the new pod can get scheduled. See the issue [#3741](https://github.com/openebs/openebs/issues/3751) for more details.
+
+### Watch Items and Known Issues - Replicated Storage
+
+- Replicated PV Mayastor does not support the capacity expansion of DiskPools as of v2.6.0.
```
Comment (avishnu) [inline docs/main/releases.md:88]: v2.7.0

Comment (balaharish7) [reply docs/main/releases.md:88]: We have to change this here as well Vishnu: https://github.com/openebs/openebs/releases/tag/v4.1.0

File: docs/versioned_docs/version-4.1.x/quickstart-guide/installation.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+```
+helm ls -n openebs
+```
+
+**Output**
+
+```
+NAME     NAMESPACE   REVISION  UPDATED                                   STATUS     CHART           APP VERSION
+openebs  openebs     1         2024-03-25 09:13:00.903321318 +0000 UTC   deployed   openebs-4.0.1   4.0.1
```
Comment (avishnu) [inline docs/versioned_docs/version-4.1.x/quickstart-guide/installation.md:97]: Shouldn't the version be 4.1.0 everywhere?

File: docs/versioned_docs/version-4.1.x/releases.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+## Watch Items and Known Issues
+
+### Watch Items and Known Issues - Local Storage
+
+Local PV ZFS / Local PV LVM on a single worker node encounters issues after upgrading to the latest versions. The issue is specifically associated with the change of the controller manifest to a Deployment type, which results in the failure of new controller pods to join the Running state. The issue appears to be due to the affinity rules set in the old pod, which are not present in the new pods. As a result, since both the old and new pods have relevant labels, the scheduler cannot place the new pod on the same node, leading to scheduling failures when there's only a single node.
+The workaround is to delete the old pod so the new pod can get scheduled. See the issue [#3741](https://github.com/openebs/openebs/issues/3751) for more details.
+
+### Watch Items and Known Issues - Replicated Storage
+
+- Replicated PV Mayastor does not support the capacity expansion of DiskPools as of v2.6.0.
```
Comment (avishnu) [inline docs/versioned_docs/version-4.1.x/releases.md]: v2.7.0

**Review by avishnu — APPROVED** [review]

**Review by niladrih — APPROVED** [review]

**Pull request merged**